src/components/annual-report/section1/useSectionOneData.js

```diff
@@ -62,7 +73,7 @@ const reportQueries = {
         params: ({ dataElements, orgUnits, currentPeriod }) => ({
             dimension: `dx:${dataElements.join(';')},ou:${orgUnits.join(
                 ';'
-            )},pe:${currentPeriod}`,
+            )},pe:${currentPeriod},co`,
```

Comment:
co is added as dimension as all data element operands will be data elements 😄 (passing `co` dimension with indicators will cause request to fail). If operand is missing coc, we will assume it's using default coc